spec.html

```diff
@@ -10067,9 +10135,11 @@ <h1>
             <dd>It is used to set the bound value of the current binding of the identifier whose name is _N_ to the value _V_. An uninitialized binding for _N_ must already exist.</dd>
           </dl>
           <emu-alg>
-            1. Assert: _envRec_ must have an uninitialized binding for _N_.
-            1. Set the bound value for _N_ in _envRec_ to _V_.
-            1. <emu-not-ref>Record</emu-not-ref> that the binding for _N_ in _envRec_ has been initialized.
+            1. Assert: _envRec_.HasBinding(_N_) is *true*.
+            1. Let _binding_ be the DeclarativeBinding in _envRec_.[[Bindings]] whose [[BoundName]] field equals _N_.
+            1. Assert: _binding_ is a SimpleDeclarativeBinding.
```

Comment:
Can't we just say `SimpleDeclarativeBinding` in place of `DeclarativeBinding` in the above step then?

Reply:
I don't think the spec does that.

That is, in a phrase like:
> the [thing] in [some list] that satisfies [some constraint]

I don't think the `[thing]` phrase is ever restrictive. Rather, it's a descriptor that's true of every element of `[some list]`. All the restriction happens in `[some constraint]`.